packages/nodes-base/nodes/Google/Sheet/v2/methods/listSearch.ts

```diff
@@ -6,91 +6,78 @@ import {
 	NodeOperationError,
 } from 'n8n-workflow';
 import { ResourceLocator } from '../helpers/GoogleSheets.types';
-import { getSpreadsheetId, sortLoadOptions } from '../helpers/GoogleSheets.utils';
-import { apiRequest, apiRequestAllItems } from '../transport';
+import { getSpreadsheetId } from '../helpers/GoogleSheets.utils';
+import { apiRequest } from '../transport';
 
 export async function spreadSheetsSearch(
 	this: ILoadOptionsFunctions,
 	filter?: string,
+	paginationToken?: string,
 ): Promise<INodeListSearchResult> {
-	try {
-		const returnData: INodeListSearchItems[] = [];
-		const query: string[] = [];
-		if (filter) {
-			query.push(`name contains '${filter.replace("'", "\\'")}'`);
-		}
-		query.push("mimeType = 'application/vnd.google-apps.spreadsheet'");
-		const qs = {
-			pageSize: 50,
-			orderBy: 'modifiedTime desc',
-			fields: 'nextPageToken, files(id, name, webViewLink)',
-			q: query.join(' and '),
-			includeItemsFromAllDrives: true,
-			supportsAllDrives: true,
```

Comment:
Don't we need includeItemsFromAllDrives and supportsAllDrives?

Reply:
thanks, fixed